doc/modules/language-guide/pages/cycles.adoc

```diff
@@ -2,11 +2,11 @@
 :proglang: Motoko
 :company-id: DFINITY
 
-Usage of the Internet Computer is measured, and paid for, in _cycles_.
-The {company-id} platform maintains a per canister balance of cycles
-and cycles can be transferred between canisters.
+Usage of the {IC} is measured, and paid for, in _cycles_.
+The {IC} maintains a balance of cycles per canister smart contract.
+In addition, cycles can be transferred between canisters.
 
-In {proglang}, each actor has a balance of cycles.
+In {proglang}, each actor, as an {IC} canister, also has a balance of cycles.
```

Comment:
"each actor, as an IC canister" I found this a bit confusing. Can I run a actor outside of IC? Why not just say "each actor" or "each IC canister".

Also, since this is Motoko documentation, I wonder if we should only say "actor" and not "IC canister". Or maybe "Motoko canister".

Reply:
```suggestion
In {proglang} programs targeting the {IC}, each actor represents an {IC} canister smart contract, and has an associated balance of cycles.
```